src/components/modules/blockManager.ts

```diff
@@ -863,7 +863,7 @@ export default class BlockManager extends Module {
     /**
      * Now using Conversion Config "import" we compose a new Block data
      */
-    let newBlockData = convertStringToBlockData(cleanData, replacingTool.conversionConfig);
+    let newBlockData = convertStringToBlockData(cleanData, replacingTool.conversionConfig, replacingTool.settings);
```

Comment:
you are passing the whole tool settings, not a config. Consider using "replacingTool.settings.config"


tools settings are this part in Editor Config

<img width="553" alt="image" src="https://github.com/user-attachments/assets/ff0564bb-38e4-4d2a-994c-afd29472c324">

Reply:
this is differnt issue, i think that this is not related to the pr problem